### ISSUE TITLE
TTL import of analyzers with nested pipelines fails

### ISSUE THREAD

**Issue by tobice** [ldvm/LDVMi#123]
At this moment, it is impossible to import any analyzer from TTL  that contains nested pipelines, such as [this one](https://github.com/ldvm/vocabulary/blob/master/rdf/examples/analyzers/ldvm-a-ruian-obce.ttl) or [this one](https://github.com/ldvm/vocabulary/blob/master/rdf/examples/analyzers/ldvm-a-ovm.ttl).

Stacktrace:

```
! @6pi173659 - Internal server error, for (POST) [/api/v1/components/add/ttl] ->

play.api.Application$$anon$1: Execution exception[[NoSuchElementException: Invoker.first]]
	at play.api.Application$class.handleError(Application.scala:296) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.api.DefaultApplication.handleError(Application.scala:402) [play_2.11-2.3.7.jar:2.3.7]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.7.jar:2.3.7]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.7.jar:2.3.7]
	at scala.Option.map(Option.scala:146) [scala-library-2.11.7.jar:na]
Caused by: java.util.NoSuchElementException: Invoker.first
	at scala.slick.jdbc.Invoker$$anonfun$first$1.apply(Invoker.scala:36) ~[slick_2.11-2.1.0.jar:na]
	at scala.slick.jdbc.Invoker$$anonfun$first$1.apply(Invoker.scala:36) ~[slick_2.11-2.1.0.jar:na]
	at scala.Option.getOrElse(Option.scala:121) [scala-library-2.11.7.jar:na]
	at scala.slick.jdbc.Invoker$class.first(Invoker.scala:36) ~[slick_2.11-2.1.0.jar:na]
	at scala.slick.jdbc.StatementInvoker.first(StatementInvoker.scala:16) ~[slick_2.11-2.1.0.jar:na]
```

The problem is on [this line](https://github.com/ldvm/LDVMi/blob/beb11ac42a50da136ce0a0bcbb84121098a357c0/src/app/model/service/impl/pipeline/PipelineServiceImpl.scala#L61) where instead of  `pipeline.componentInstances.last.componentInstance.uri` should be `pipeline.componentInstances.last.componentInstance.templateUri`.

**Comment by jirihelmich** [comment]
Fixed by a3f69d2.

**Comment by jirihelmich** [comment]
Hint: if the commit message was "Fixes #123", the issue would close itself after merging the PR. This is now being automatically deployed and will be live in a few minutes.

Thanks for the fix.
